HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -428,6 +428,78 @@ def self.apply_hvac(hpxml, weather, convert_shared_systems)
       HVAC.apply_shared_systems(hpxml)
     end
 
+    # HVAC efficiencies (based on HEScore assumption)
+    hpxml.heating_systems.each do |heating_system|
+      year_installed = heating_system.year_installed
+      heating_system_type = heating_system.heating_system_type
+      heating_system_fuel = heating_system.heating_system_fuel
+
+      if [HPXML::HVACTypeBoiler, HPXML::HVACTypeFurnace, HPXML::HVACTypeWallFurnace, HPXML::HVACTypeFloorFurnace].include? heating_system_type
+        next unless heating_system.heating_efficiency_afue.nil?
+
+        if heating_system_fuel == HPXML::FuelTypeElectricity
+          heating_system.heating_efficiency_afue = 0.98
+        else
+          heating_system.heating_efficiency_afue = HVAC.get_default_hvac_efficiency(year_installed, heating_system_type, heating_system_fuel, HPXML::UnitsAFUE)
+        end
+        heating_system.heating_efficiency_afue_isdefaulted = true
+      elsif [HPXML::HVACTypeElectricResistance].include? heating_system_type
+        next unless heating_system.heating_efficiency_percent.nil?
+
+        heating_system.heating_efficiency_percent = 0.98
```

Comment:
From my perspective, this is the only question holding up the PR. We have sample files where we assume 1.0 but here we default to 0.98 (per HEScore). We either need to update the defaults to 1.0 instead of 0.98 or we should update our sample files to use 0.98 instead, as we are currently inconsistent.

I'm not sure what to recommend yet, will think about it some more...

Reply:
Okay, I suggest we use 0.98 for furnaces/boilers and 1.0 for electric resistance, stoves, space heaters, etc. Please update our defaults, HPXML sample files, docs, and anything else (?) to be consistent with that.